docs/examples/index.html

```diff
@@ -85,3 +85,22 @@ <h3>Templates</h3>
     </nav>
   </div>
 </div>
+<div class="row">
+  <div class="col-12">
```

Comment:
We normally limit text blocks to `col-8` could you change this to help if edited in the future.